graviti/openapi/data.py

```diff
@@ -177,3 +177,65 @@ def list_commit_data(
     return _list_data(
         access_key, url, columns=columns, order_by=order_by, offset=offset, limit=limit
     )
+
+
+def update_data(
+    access_key: str,
+    url: str,
+    owner: str,
+    dataset: str,
+    *,
+    draft_number: int,
+    sheet: str,
+    data: List[Dict[str, Any]],
+    offset: int = 0,
+    order_by: Optional[str] = None,
+) -> None:
+    """Execute the OpenAPI `PATCH /v2/datasets/{owner}/{dataset}/drafts/{draft_number}\
+    /sheets/{sheet}/data`.
+
+    Arguments:
+        access_key: User's access key.
+        url: The URL of the graviti website.
+        owner: The owner of the dataset.
+        dataset: Name of the dataset, unique for a user.
+        draft_number: The draft number.
+        sheet: The sheet name.
+        data: The update data.
+        offset: The starting position of the update data.
+        order_by: The string determine the order of sorting by  the order in
+            which column names appear.
+
+    Examples:
+        >>> update_dataset(
+        ...     "ACCESSKEY-********",
+        ...     "https://api.graviti.com/",
+        ...     "czhual",
+        ...     "OxfordIIITPet",
+        ...     draft_number = 1,
+        ...     sheet = "train",
+        ...     data = [
+        ...                 {
+        ...                     "filename": "0000f77c-6257be58.jpg",
+        ...                     "image": {
+        ...                         "url": "https://content-store-prod-vers",
+        ...                         "checksum": "dcc197970e607f7576d978972f6fb312911ce005",
+        ...                         "type": "remoteFile"
+        ...                     },
+        ...                     "attribute": {
+        ...                         "weather": "clear",
+        ...                         "scene": "city street",
+        ...                         "timeofday": "daytime"
+        ...                     },
+        ...                 },
+        ...                 ...(total 5 items)
```

Comment:
in the example, use the right code,  do not use`...(total 5 items)`. we could write two rows here